mri/operators/fourier/utils.py

```diff
@@ -143,8 +143,8 @@ def get_stacks_fourier(kspace_loc, volume_shape):
     # Find the mask used to sample stacks in z direction
     full_stack_z_loc = convert_mask_to_locations(
         np.ones(volume_shape[2]),
-    )[:, 0]
-    sampled_stack_z_loc = np.unique(kspace_loc[:, 2])
+    )[:, 0].astype('float16')
+    sampled_stack_z_loc = np.unique(kspace_loc[:, 2]).astype('float16')
```

Comment:
I am a bit apprehensive about why we moved to float16 for comparision. If the earlier code was not working, it would be better to change the threshold for comparision in `np.isclose` below.

Reply:
Agree with that, I'll have a look to change the threshold